install.psm1

```diff
@@ -0,0 +1,262 @@
+﻿#
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#Requires -RunAsAdministrator
+
+function Get-Install-Directory([string]$InstallDir) {
+    $dir = "OpenTelemetry .NET AutoInstrumentation"
+    
+    if($InstallDir -eq "<auto>" -or $installDir -eq "AppData") {
```

Comment:
Can you reformat the code? There should be spaces after `if`, `for` ,`=`

Reference https://poshcode.gitbook.io/powershell-practice-and-style/style-guide/code-layout-and-formatting

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1297/commits/a25ccb1da14bd4c8bf65278dfa923d8267358fdd

PowerShell ISE seems not to care about format, fixed with VS Code.